tests/test_globalization.py

```diff
@@ -14,25 +14,19 @@
 import mssqlcli.completion_refresher as completion_refresher
 
 
-"""
-All tests require a live connection to test server.
-Make modifications to mssqlutils.create_mssql_cli_client()
-to use a different server and database.
-Please Note: These tests cannot be run offline.
-"""
+# All tests require a live connection to test server.
+# Make modifications to mssqlutils.create_mssql_cli_client()
+# to use a different server and database.
+# Please Note: These tests cannot be run offline.
 
 
-class GlobalizationTests():
+class GlobalizationTests:
     @staticmethod
     def run_query(mssqlcliclient, query):
         success = True
-        try:
-            for _, _, _, _, is_error in mssqlcliclient.execute_query(query):
-                if is_error is True:
-                    success = False
-                    break
-        except:
-            success = False
+        for _, _, status, _, is_error in mssqlcliclient.execute_query(query):
```

Comment:
OK so this captures additional details from the call now and logs in the exception stack?

Reply:
This and the change to main.py are the only 2 required to identify why the test fails, correct?

Reply:
Yes, all correct.